manifests/0000_70_dns-operator_00-cluster-role.yaml

```diff
@@ -111,3 +111,27 @@ rules:
   - clusteroperators/status
   verbs:
   - update
+
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - featuregates
```

Comment:
Are you sure this is needed ?  Featuregates are normally accessible, why are these different?

Reply:
FeatureGates also are like any other objects in k8s and the access needs to be added to the cluster role. It is added in cluster ingress operator as well: https://github.com/arkadeepsen/cluster-ingress-operator/blob/9a872878deb0421105da88c9fe25b1079aa2f2ea/manifests/00-cluster-role.yaml#L115